skopt/space/space.py

```diff
@@ -550,6 +552,49 @@ def is_real(self):
         """
         return all([isinstance(dim, Real) for dim in self.dimensions])
 
+    @classmethod
+    def from_yaml(cls, yml_path, namespace=None):
+        """Create Space from yaml configuration file
+
+        Parameters
+        ----------
+        * `yml_path` [str]:
+           Full path to yaml configuration file
+        * `namespace` [str, default=None]:
+           Namespace within configuration file to use
+
+        Returns
+        -------
+        * `space` [Space]:
+           Instantiated Space object
+        """
+        with open(yml_path, 'rb') as f:
+            cfg = yaml.load(f)
```

Comment:
where possible i'd spell out variables, so `cfg` -> `config` and so on

Reply:
sure no problem